tests/foreman/cli/test_repository_set.py

```diff
@@ -56,21 +52,19 @@ def test_repositoryset_available_repositories(self):
             u'organization-id': org['id'],
             u'product': rhel_product_name,
         })
-        self.assertEqual(result.return_code, 0)
         self.assertEqual(
-            sum(int(repo['enabled'] == u'true') for repo in result.stdout),
-            0
+            sum(int(repo['enabled'] == u'true') for repo in result),
+            0,
```

Comment:
There is no need to add comma at the end of functions.  This comment applies to the entire file